README.md

```diff
@@ -7,29 +7,54 @@
 
 > Share your conferences activity to your buddies
 
+## Requirements
+
+This project requires :
+
+- Docker 
+- mkcert (https://github.com/FiloSottile/mkcert)
+- pipenv (https://github.com/pypa/pipenv)
+
 ## Installation
 
 ### Local
 
-> Add `local.starfleet.app` to your hosts file
+First, you will need to add local.starfleet.app to your hosts file.
+Open it with :
+```
+sudo nano /etc/hosts
+```
 
-Use [mkcert](https://github.com/FiloSottile/mkcert) to install a CA on your system to be able to generate a valid self-signed certificate for local.starfleet.app
+Then paste in `127.0.0.1 local.starfleet.app`
 
-`$ mkcert *.starfleet.app`
+You now need to generate a certificate for `local.starfleet.app`. This is why you need mkcert, which you may find here with the installation instructions : https://github.com/FiloSottile/mkcert
```

Comment:
1. There are never space before double colon in English.
2. The sentence if too long IHMO

```
You need [mkcert](https://github.com/FiloSottile/mkcert) to generate a certificate for `local.starfleet.app`:
```